lib/tools/system-calls.js

```diff
@@ -75,4 +76,112 @@ systemCallMethods.getBinaryFromPath = async function(binaryName) {
   }
 };
 
+systemCallMethods.getConnectedDevices = async function () {
+  log.debug("Getting connected devices...");
+  try {
+    let {stdout} = await exec(this.adb.path, ['devices']);
```

Comment:
Ah! Thanks. I knew about object destructuring, but didn't clue in to the pattern here.